### DISCUSSION TITLE
Css not load properly or fast on my site awesmaticfrontend.vercel.app

### DISCUSSION THREAD

**Discussion by naveenkhatanagit** [vercel/next.js#50151] [Help]
### Summary

Css not load properly or fast on my site awesmaticfrontend.vercel.app. i have use ssr  and image component with placeholder loader 

### Example

awesmaticfrontend.vercel.app

**Comment by icyJoseph** [comment]
How are you loading your stylesheets? Could you share the `_document` and `_app` of your project. 

Your page's HTML ends with:

```html
</script></body><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"/><script src="https://kit.fontawesome.com/a46d0a2aba.js" crossorigin="anonymous"></script><link rel="stylesheet" href="[https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css ](https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css)"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.5.9/slick-theme.min.css"/><script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script></html></body></html>
```

Which is really weird. We see the body tag ends, and then a bunch of links follow, but then we close the html tag, a body tag again, and an html tag again? It must be something we are doing wrong in `_document` or `_app`, and most likely the root of your issue.

**Comment by naveenkhatanagit** [comment]
_app.js
import '@/styles/globals.css'
import '@/styles/style.css'
import '@/styles/resposive.css'
import { SessionProvider } from "next-auth/react"
import LayoutComponent from '@/components/Layouts/LayoutComponent'
import { Provider } from "../context";
import NProgress from 'nprogress'; //nprogress module
import Router from 'next/router';
import '@/styles/ngprogress.css'

Router.events.on('routeChangeStart', () => NProgress.start()); Router.events.on('routeChangeComplete', () => NProgress.done()); Router.events.on('routeChangeError', () => NProgress.done());
export default function App({
  Component,
  pageProps: { session, ...pageProps },
}) {
  return <SessionProvider session={session}>
    <Provider>
      <LayoutComponent>
        <Component {...pageProps} />
      </LayoutComponent>
    </Provider>
  </SessionProvider>
}
_document.js
import { Html, Head, Main, NextScript } from 'next/document'

export default function Document() {
  return (
    <Html lang="en">
      <Head />
      <body>
        <Main />
        <NextScript />
      </body>
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"/>
    <script src="https://kit.fontawesome.com/a46d0a2aba.js" crossOrigin="anonymous"></script>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css
    "/>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.5.9/slick-theme.min.css"/>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
    </Html>
  )
}

**Comment by naveenkhatanagit** [comment]
yes solved thanks

**Reply by icyJoseph** [reply]
What was the issue? Was it that your `_document` file is/was incomplete?
